pubspec.yaml

```diff
@@ -25,7 +25,7 @@ dependencies:
     sdk: flutter
   http: ^0.12.2
   flutter_staggered_grid_view: ^0.3.3
-
+  rxdart: ^0.25.0
```

Comment:
Im not really sure if we are using this on the project, can we remove it ?

Reply:
I need to use it in genres_bloc.